packages/generator/src/action-function-import/function.ts

```diff
@@ -51,37 +51,18 @@ function getFunctionImportStatements(
       }, 'const params = {\n') + '\n}'
     : '{}';
 
-  const parameters =
-    service.oDataVersion === 'v2'
-      ? getFunctionImportRequestBuilderArgumentsV2(functionImport, service)
-      : getFunctionImportRequestBuilderArgumentsV4(functionImport, service);
+  let parameters = getRequestBuilderArgumentsBase(
```

Comment:
I can't fully wrap my head around this - `getRequestBuilderArgumentsBase` returns `string[]` and the last  item in the returned list is the first argument passed into the function? So the last item would be `'params'` in this case here?
Probably I'm just reading it wrong, but if not, I would argue that this should not be necessary.